src/Elements/Bundle/AlternateObjectTreesBundle/LevelDefinition/Input.php

```diff
@@ -52,15 +52,19 @@ public function getGroupedValues($condition, $offset = null, $limit = null)
         // create condition
         $where = $this->condition == '' ? 1 : $this->condition;
         if ($condition) {
+            $where = 'LIKE \'%'.$where.'%\'';
```

Comment:
I strongly advise to escape `$where` at least in some way.
Please take a look at either Doctrine's `Connection::quote()` (if doctrine is available and can be injected):
https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/security.html#right-quoting-escaping-values
Or PDO::quote():
https://www.php.net/manual/en/pdo.quote.php

Reply:
updated, thanks for the tip!